ggml/src/ggml-cuda/common.cuh

```diff
@@ -316,6 +318,10 @@ static bool cp_async_available(const int cc) {
     return GGML_CUDA_CC_IS_NVIDIA(cc) && ggml_cuda_highest_compiled_arch(cc) >= GGML_CUDA_CC_AMPERE;
 }
 
+static bool blackwell_mma_available(const int cc) {
+    return GGML_CUDA_CC_IS_NVIDIA(cc) && ggml_cuda_highest_compiled_arch(cc) >= GGML_CUDA_CC_BLACKWELL;
```

Comment:
We are using instructions currently available to `120f`, so anything between 1200 and 1299. There is no guarantee that CC 13 will support those instructions as well, so I'd recommend we check against the aforementioned range